interactive_engine/executor/ir/graph_proxy/src/apis/graph/mod.rs

```diff
@@ -174,4 +174,12 @@ impl QueryParams {
             None
         }
     }
+
+    // is_queryable doesn't consider tables as we assume that the table info can be inferred directly from current data.
+    pub fn is_queryable(&self) -> bool {
+        !(self.filter.is_none()
+            && self.limit.is_none()
```

Comment:
Why we need to consider limit and sample_ratio then. And if this is the case, directly using `! ( self.filter.is_none() && self.columns.is_none())` without introducing this arbitrary function is a better idea.

Reply:
Done